darts/models/forecasting/prophet_model.py

```diff
@@ -110,6 +116,14 @@ def _fit(self, series: TimeSeries, future_covariates: Optional[TimeSeries] = Non
         fit_df = pd.DataFrame(
             data={"ds": series.time_index, "y": series.univariate_values()}
         )
+        if self.is_logistic:
+            raise_if(
```

Comment:
Could we receive `cap` and `floor` as (optional) arguments of `__init__` instead? I think it would be cleaner as it would allow checking for their presence directly whenever a model is instantiated with logistic growth. I also think it'd make sense to see them as model hyperparameters and so specifying them in the constructor is natural I think. Finally, it would remove the need for users to have to know about `set_capacity()`. But let me know if you think I missed something.

Reply:
There were some concerns about adding them as optional arguments in #178, which is I think why this issue #628 suggested using a separate function. 
But I agree, I also think it would be cleaner to have `cap` and `floor` as arguments to `__init__`.
Shall I change it?